### ISSUE TITLE
Is there documentation or a guide available for use?

### ISSUE THREAD

**Issue by The-Ladle** [Turasa/libsignal-service-java#36]
I came across this and am very curious if this has documentation for use.

**Comment by Thireus** [comment]
I was going to ask the same question today. I'm having quite a hard time understanding how to properly use this library. Examples would be welcomed.

**Comment by AsamK** [comment]
No, it's developed as an internal library for signal Android app, without additional documentation. 
You can look at the source code of Signal Android or signal-cli, to see how it can be used.
